src/agent/src/rpc.rs

```diff
@@ -90,6 +93,8 @@ const ERR_INVALID_BLOCK_SIZE: &str = "Invalid block size";
 const ERR_NO_LINUX_FIELD: &str = "Spec does not contain linux field";
 const ERR_NO_SANDBOX_PIDNS: &str = "Sandbox does not have sandbox_pidns";
 
+const IPTABLES_RESTORE_WAIT_SEC: &str = "5";
```

Comment:
How did we arrive at the 5s wait time? Could we either add a comment describing the choice of 5 or make this a configurable parameter?

Reply:
Instead of using --wait, consider taking an explicit flock that you control. That way you know that nothing else is active (or you know that something else is active). relying on iptables's lock is opaque.

Reply:
This is interesting. I think that a lock I own assumes the other potential writer would also be using this lock (not realistic).  I think we can utilize a sandbox lock on host side to ensure that we don't have multiple instances of get/set/get running at a time, but still rely on the xtable lock in the guest? WDYT?

Reply:
in that case do not use the wait and fail right away.